tlatools/org.lamport.tlatools/src/tla2sany/explorer/Explorer.java

```diff
@@ -170,7 +170,7 @@ private void lookUpAndPrintDef(String symbName) {
 			} else if (sym instanceof FormalParamNode) {
 				System.out.print("Module: " + ((FormalParamNode) sym).getModuleNode().getName());
 			}
-			System.out.println(((ExploreNode) (symbolVect.elementAt(i))).toString(100));
+			System.out.println(((ExploreNode) (symbolVect.elementAt(i))).toString(100, errors));
```

Comment:
Consider logically merging the subsequent `System.out.println();` statement into this line.